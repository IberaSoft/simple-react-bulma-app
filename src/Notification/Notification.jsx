import React, { Component } from 'react';

class Notification extends Component {
    state = {
        isHidden: false
    }

    handleClick = (value) => {
        this.setState({ isHidden: value })
    }

    render() {
        const { activeIndex } = this.props

        return (
            <div className={`notification is-info ${(this.state.isHidden === true ) ? 'is-hidden' : ''}`}>
                <button className="delete" onClick={() => this.handleClick(true)}></button>
                <p>The active index is: {activeIndex}</p>
            </div>
        )
    }
}

export default Notification;