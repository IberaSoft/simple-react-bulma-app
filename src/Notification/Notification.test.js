import React from 'react'
import Notification from './Notification'
import renderer from 'react-test-renderer';

describe('Notification component', () => {
    it('Should match the snapshot', () => {
        const wrapper = renderer
            .create(<Notification />)
            .toJSON();
        expect(wrapper).toMatchSnapshot();
    });
})