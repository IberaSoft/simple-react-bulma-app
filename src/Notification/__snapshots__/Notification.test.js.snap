// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification component Should match the snapshot 1`] = `
<div
  className="notification is-info "
>
  <button
    className="delete"
    onClick={[Function]}
  />
  <p>
    The active index is: 
  </p>
</div>
`;
